src/Microsoft.AspNetCore.Server.Testing/Deployers/RemoteWindowsDeployer/StartServer.ps1

```diff
@@ -28,19 +31,28 @@ IF (-Not [string]::IsNullOrWhitespace($environmentVariables))
 	}
 }
 
+# Temporary workaround for issue https://github.com/dotnet/cli/issues/2967
+if ($executablePath -ne "dotnet.exe"){
+    $destinationDir = Split-Path $executablePath
+    Copy-Item C:\Windows\System32\forwarders\shell32.dll $destinationDir
+}
+
+$command = $executablePath + " " + $executableParameters + " --server.urls " + $applicationBaseUrl
```

Comment:
Currently the script doesn't handle paths having spaces within them. I will file a separate issue for it.